examples/pulp_varnames.py

```diff
@@ -0,0 +1,138 @@
+from cornflow_client import CornFlow, group_variables_by_name
```

Comment:
Maybe we should move this example to the new repository

Reply:
I think maybe we should have a `requirements-dev.txt` for cornflow that includes `cornflow_client` so it can be tested. This way we can keep the tests in cornflow.

Reply:
Ok, as it can be useful on development but it should not be needed on deployment